blockly/blocks/base.js

```diff
@@ -196,3 +196,56 @@ Blockly.Blocks['serial_print'] = {
     this.setTooltip('Prints data to the console/serial port as human-readable ASCII text.');
   }
 };
+
+Blockly.Blocks['lcd_setup'] = {
+  init:function(){
```

Comment:
Code across various branches will be a lot easier to read if we keep to a consistent coding style - including putting spaces between 'init' and 'function, indenting continued lines ('.appendField' etc).